newton/tests/test_import_usd.py

```
@@ -2424,6 +2424,73 @@ def PhysicsRevoluteJoint "Joint2"
         self.assertTrue(np.any(jnt_actgravcomp))
         self.assertTrue(np.any(~jnt_actgravcomp))
 
+    @unittest.skipUnless(USD_AVAILABLE, "Requires usd-core")
+    def test_option_impratio_parsing(self):
+        """Test parsing of impratio from USD PhysicsScene with mjc:option:impratio attribute."""
+        from pxr import Usd  # noqa: PLC0415
```

Ruff (0.14.8)

Unused `noqa` directive (non-enabled: `PLC0415`)

Remove unused `noqa` directive

(RUF100)